src/test/ui/regions-fn-subtyping-return-static.stderr

```diff
@@ -6,8 +6,6 @@ error[E0308]: mismatched types
    |
    = note: expected type `fn(&'cx S) -> &'cx S`
               found type `fn(&'a S) -> &S {bar::<'_>}`
-   = note: lifetime parameter `'b` declared on fn `bar` appears only in the return type, but here is required to be higher-ranked, which means that `'b` must appear in both argument and return types
-   = note: this error is the result of a recent bug fix; for more information, see issue #33685 <https://github.com/rust-lang/rust/issues/33685>
```

Comment:
Yeah, so I think I kept a lot of that machinery precisely so that we could continue to generate these notes -- or at least the first one -- but it is a lot of "stuff" for a rather obscure corner case.